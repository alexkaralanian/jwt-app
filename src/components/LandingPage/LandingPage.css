.centeredContainer {
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
}

.innerContainer {
  max-width: 500px;
}

.largeHeader {
  color: #4a90e2;
  font-weight: 100;
  font-size: 80px;
  margin-top: 80px;
  text-align: center;
}
.subHeader {
  color: #4a90e2;
  font-size: 25px;
  text-align: center;
  line-height: 40px;
  font-style: italic;
}
