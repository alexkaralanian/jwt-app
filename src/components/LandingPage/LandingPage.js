import React from "react";
import "./LandingPage.css";

const LandingPage = () => (
  <div className="centeredContainer">
    <div className="innerContainer">
      <h1 className="largeHeader">JWT</h1>
      <h3 className="subHeader">
        Example React Redux App Using JSON Web Token Authentication
      </h3>
    </div>
  </div>
);

export default LandingPage;
