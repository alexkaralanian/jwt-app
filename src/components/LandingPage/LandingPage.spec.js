import React from "react";
import { shallow, mount } from "enzyme";
import LandingPage from "./LandingPage";

test("LandingPage renders without crashing", () => {
  mount(<LandingPage />);
});

test("Landing Page Component renders correctly", () => {
  const component = shallow(<LandingPage />);
  expect(component).toMatchSnapshot();
});
