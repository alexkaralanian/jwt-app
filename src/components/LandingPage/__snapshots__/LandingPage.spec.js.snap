// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Page Component renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="centeredContainer"
>
    <div
        className="innerContainer"
    >
        <h1
            className="largeHeader"
        >
            JWT
        </h1>
        <h3
            className="subHeader"
        >
            Example React Redux App Using JSON Web Token Authentication
        </h3>
    </div>
</div>,
  "nodes": Array [
    <div
      className="centeredContainer"
>
      <div
            className="innerContainer"
      >
            <h1
                  className="largeHeader"
            >
                  JWT
            </h1>
            <h3
                  className="subHeader"
            >
                  Example React Redux App Using JSON Web Token Authentication
            </h3>
      </div>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <LandingPage />,
      "_debugID": 7,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="centeredContainer"
>
          <div
                    className="innerContainer"
          >
                    <h1
                              className="largeHeader"
                    >
                              JWT
                    </h1>
                    <h3
                              className="subHeader"
                    >
                              Example React Redux App Using JSON Web Token Authentication
                    </h3>
          </div>
</div>,
        "_debugID": 8,
        "_renderedOutput": <div
          className="centeredContainer"
>
          <div
                    className="innerContainer"
          >
                    <h1
                              className="largeHeader"
                    >
                              JWT
                    </h1>
                    <h3
                              className="subHeader"
                    >
                              Example React Redux App Using JSON Web Token Authentication
                    </h3>
          </div>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <LandingPage />,
}
`;
